azurelinuxagent/pa/rdma/suse.py

```diff
@@ -93,8 +93,7 @@ def install_driver(self):
                 msg = 'RDMA: Successfully installed "%s" from '
                 msg += 'configured repositories'
                 logger.info(msg % complete_name)
-                self.load_driver_module()
-                if requires_reboot:
+                if not self.load_driver_module() or requires_reboot:
```

Comment:
Actual I mean the 2nd reboot (not the next reboot: 1st reboot). The 1st reboot is needed to properly load (or attempt to load) the driver. The 2nd reboot will not happen because the driver package is already installed.